<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PosInputBox</class>
 <widget class="QWidget" name="PosInputBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PosInputBox</string>
  </property>
  <widget class="QGroupBox" name="inputBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	color: white;
}

QSpinBox:: color: color: rgb(0, 0, 0);
</string>
   </property>
   <property name="title">
    <string>Вход</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="XYZAp">
     <item>
      <widget class="QSpinBox" name="XApBox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0)</string>
       </property>
       <property name="minimum">
        <number>-100000000</number>
       </property>
       <property name="maximum">
        <number>1000000000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ZApBox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0)</string>
       </property>
       <property name="minimum">
        <number>-100000000</number>
       </property>
       <property name="maximum">
        <number>1000000000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HeightApBox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0)</string>
       </property>
       <property name="minimum">
        <number>-100000</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>144</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ParamAp">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="KursApBox">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0)</string>
         </property>
         <property name="minimum">
          <number>-100000</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="singleStep">
          <number>30</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="AngApBox">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0)</string>
         </property>
         <property name="minimum">
          <number>-100000</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSpinBox" name="SpeedAprBox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0)</string>
       </property>
       <property name="minimum">
        <number>-100000</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>150</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Цель</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>       ЛА</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>144</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ParamTar">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="KursTarBox">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0)</string>
         </property>
         <property name="minimum">
          <number>-100000</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="singleStep">
          <number>30</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="AngTarBox">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0)</string>
         </property>
         <property name="minimum">
          <number>-100000</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSpinBox" name="SpeedTarBox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0)</string>
       </property>
       <property name="minimum">
        <number>-100000</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>149</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>111</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="xlabel">
       <property name="text">
        <string>               X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Zlabel">
       <property name="text">
        <string>               Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Heightlabel">
       <property name="text">
        <string>               H</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="anglelabel">
       <property name="text">
        <string>        Курс, Наклон</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="spdlabel">
       <property name="text">
        <string>         Скорость</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="XYZTar">
     <item>
      <widget class="QSpinBox" name="XTarBox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0)</string>
       </property>
       <property name="minimum">
        <number>-100000000</number>
       </property>
       <property name="maximum">
        <number>1000000000</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
       <property name="value">
        <number>85000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ZTarBox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0)</string>
       </property>
       <property name="minimum">
        <number>-100000000</number>
       </property>
       <property name="maximum">
        <number>100000000</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
       <property name="value">
        <number>-300000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HeightTarBox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0)</string>
       </property>
       <property name="minimum">
        <number>-100000</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
       <property name="value">
        <number>12000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
